wordcloud/wordcloud.py

```diff
@@ -233,8 +233,8 @@ def fit_words(self, words):
         self.layout_ = list(zip(words, font_sizes, positions, orientations, colors))
         return self.layout_
 
-    def process_text(self, text):
-        """Splits a long text into words, eliminates the stopwords.
+    def _get_words(self, text):
```

Comment:
Thanks for pointing out. I don't know that until you told me, as I am only modifying to my own need.